src/main/resources/doc/en/html/libs/mem/dualportram.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="created" content="2025-06-21T14:47:54.000000">
+    <meta name="changed" content="2025-06-21T18:26:34.532727">
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <meta http-equiv="Content-Language" content="en">
+    <title>
+      Dual-Port RAM&lt;
+    </title>
+    <link rel="stylesheet" type="text/css" href="../../style.css">
+  </head>
+  <body>
+    <div class="maindiv">
+      <h1>
+        <!-- TODO: change Dual-Port RAM's icon? -->
+        <img class="iconlibs" src="../../../../icons/6464/ram.png" alt="#########" width="32" height="32"> <em>Dual-Port RAM</em>
+      </h1>
+      <table>
+        <tbody>
+          <tr>
+            <td>
+              <strong>Library:</strong>
+            </td>
+            <td>
+              <a href="index.html">Memory</a>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <strong>Introduced:</strong>
+            </td>
+            <td>
+              <!-- TODO: placeholder, change before release -->
+              T.B.D.
+            </td>
+          </tr>
+          <tr>
+            <td valign="top">
+              <strong>Appearance:</strong>
+            </td>
+            <td valign="top">
+              <img class="appearancelibs" src="../../../../img-libs/dpram_appearance.png" alt="#########" width="64" height="64">
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h2>
+        Behavior
+      </h2>
+      <p>
+        The Dual Port RAM component is a special memory component that behaves similarly to the RAM, storing up to 16,777,216 values (specified in the Address Bit Width attribute), each of which can include up to to 32 bits (specified in the Data Bit Width attribute).

```

Comment:
Isn't the data bit width not limited to 64 bits?

Regarding the maximum size, PR #2291 might have an impact to this PR if it should get finished/merged at some point.